syntax = "proto3";

package protobuf.messages.base_msg;

message status {
    string status = 1;
}

message command {
    string cmd = 1;
}

message get_request {
    string req = 1;
}

message vector3 {
    float x = 1;
    float y = 2;
    float z = 3;
}

message vector2 {
    float x = 1;
    float y = 2;
}

